apps/builder/app/builder/shared/tree/index.tsx

```diff
@@ -98,3 +104,10 @@ export const InstanceTreeNode = (
     renderItem={instanceRelatedProps.renderItem}
   />
 );
+
+export const getInstanceLabel = (
+  instance: { label?: string },
+  meta: WsComponentMeta
+) => {
+  return instance.label ? humanizeString(instance.label) : meta.label;
```

Comment:
Do we need to "humanize" user input? Can be confusing.

Reply:
Idk, let me check with figma and others, would be really nice to have consistent naming without working hard to do so

Reply:
removed